### PR TITLE
Change flask_configfile to TOML

### PR DESCRIPTION
**Pull request by cskaandorp**
This PR enables users to config the ASReview webapp with a TOML file. Please note: Flask 2.3.0 and up is needed. Also note that toml can be parsed in Python 3.11 with a std lib called tomllib. Versions below 3.11 must use its predecessor tomli, hence the try/except import in start_flask. On my end flake8 is satisfied.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def get_long_description():
     "tqdm",
     "gevent>=20",
     "datahugger>=0.2",
+    "tomli",  # included in Python 3.11 as tomllib
```
Comment (J535D165) [inline setup.py:66]: See the syntax below for importlib_metadata.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Nice!

File: setup.py (hunk 1 above)
Comment (cskaandorp) [reply setup.py:66]: Done.

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks!!

File: setup.py (hunk 1 above)
Comment (J535D165) [reply setup.py:66]: But where?

**Pull request merged**